assets/js/googlesitekit/widgets/util/get-widget-component-props.js

```diff
@@ -54,7 +54,7 @@ export const getWidgetComponentProps = memize(
 	}
 );
 
-function withWidgetSlug( widgetSlug ) {
+export const withWidgetSlug = ( widgetSlug ) => {
```

Comment:
This change shouldn't be necessary per my other comment.

Reply:
Reverted the change @aaemnnosttv